packages/api/lambdas/sf-event-sqs-to-db-records/index.js

```diff
@@ -145,7 +158,13 @@ const handler = async (event) => {
       return await writeRecords({ ...event, cumulusMessage, knex });
     } catch (error) {
       log.error(`Writing message failed: ${JSON.stringify(message)}`, error);
-      return sendSQSMessage(process.env.DeadLetterQueue, message);
+      return sendSQSMessage(
```

Comment:
This is throwing a typing error due to the parameters in sendSQSMessage being incorrect (the third param is optional)